manifests/w/WiresharkFoundation/Wireshark/3.6.8/WiresharkFoundation.Wireshark.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: WiresharkFoundation.Wireshark
+PackageVersion: 3.6.8

```

```suggestion
PackageVersion: 3.6.8.0
```
This should match Control Panel